scripts/development/m3_stack/start_m3.sh

```diff
@@ -183,7 +186,7 @@ echo "Validating topology"
 echo "Done validating topology"
 
 echo "Waiting until shards are marked as available"
-ATTEMPTS=10 TIMEOUT=2 retry_with_backoff  \
+ATTEMPTS=100 TIMEOUT=2 retry_with_backoff  \
```

Comment:
Yes! Much nicer haha, can we do the same for integration tests?

Reply:
True yeah, will update the common ones.